include/sparrow/layout.hpp

```diff
@@ -120,13 +124,18 @@ namespace sparrow
         const_iterator cbegin() const;
         const_iterator cend() const ;
```

Comment:
For consistency with the itetrator names, thes methods should be renamed `value_begin` / `value_end` and `value_cbegin` / `value_cend`.